group_vars/all.yml

```diff
@@ -118,7 +118,7 @@ inspection_iprange: 192.168.24.110,192.168.24.250
 # undercloud to access overcloud resources
 external_gateway: 172.17.5.1/24
 external_network_vlan_id: 300
-clean_debug: False
+clean_nodes: True
```

Comment:
can you change this on CI vars also so you will get a green mark from CI ;-)

Reply:
Sure, wasn't sure if you wanted node cleaning in CI :)